aptos-move/move-examples/marketplace/README.md

```diff
@@ -0,0 +1,32 @@
+This introduces the core for a potential Aptos standard around marketplace for assets on-chain.
+
+The goals of this package are to
+* Separate core logical components for readability and expansion over time. Eliminate data from being the layer of compatibility and instead the exposed functions.
```

Comment:
"Eliminate data from being the layer of compatibility and instead the exposed functions." doesn't make sense to me at all. Can you paraphrase or elaborate more?